lib/open_mic.rb

```diff
@@ -0,0 +1,15 @@
+require 'pry'
+class OpenMic
```

Comment:
For the future, try to finish iteration 3 before moving on to iteration 4 - there is one method in user still to be created: `joke_by_id`